.changeset/wise-forks-play.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/backend-common': minor
```

Comment:
tbh I think we should avoid breaking changes in `backend-common`, just moving over the deprecated implementations to then be deleted after the next release

Reply:
Alright, never mind let's ship this to avoid a messy refactor 😅